applicationconfig/client/README.md

```diff
@@ -0,0 +1,84 @@
+# App Configuration client library for Java
+Azure App Configuration allows developers to store, retrieve and manage access to application settings all in one place. It is easy to set up and simple to use from any application. It gives developers the ability to modify an application's behavior on demand without having to redeploy the application.
```

Comment:
This doesn't follow the [Readme template](https://github.com/Azure/azure-sdk/blob/master/docs/README-TEMPLATE.md) . The service description should be taken from: https://github.com/Azure/azure-sdk/blob/master/docs/README-TEMPLATE.md